### ISSUE TITLE
Able to switch tabs but Lockdown still forces processes to end

### ISSUE THREAD

**Issue by kiwi08522** [Totsukawaii/UndownUnlock#13]
LDB is still asking me to end certain processes after DLL is injected and I can switch tabs fine. Wondering if this is a bug or normal?

**Comment by Totsukawaii** [comment]
This is normal just click close, the process won't actually close.

**Comment by kiwi08522** [comment]
@Totsukawaii It'a definitely closing the process. I have it open & it shuts down. I've tried closing Lockdown & seeing if it actually closed & it does unfortunately.

**Comment by Totsukawaii** [comment]
what process is it? Task Manager for example does not close. If Task Manager closes for you the dll has not been injected, I suggest you repeat the installation steps.

**Comment by kiwi08522** [comment]
RealVNC. I'll check if Task Manager closes. All looks good on both controller & injector, and I can change tabs, etc, so I think installation is OK.

**Comment by kiwi08522** [comment]
Task Manager did close. I was still able to switch tabs & injector said it injected successfully.

**Comment by Totsukawaii** [comment]
![image](https://github.com/Totsukawaii/UndownUnlock/assets/45559904/1dce3fc9-0001-4877-b799-f6245d9c9b9d)
do you get this pop up after launching LDB?

**Comment by kiwi08522** [comment]
No. But the command line does say it injected & I'm able to switch tabs.

**Comment by Totsukawaii** [comment]
Please repeat the installation steps to install Miniconda correctly. Your environment has not been step up correctly.

**Comment by mk6398** [comment]
@kiwi08522 what version of LDB are you running?

**Comment by kiwi08522** [comment]
@Totsukawaii I repeated installation steps on a new Windows install & getting the same issue. It's installing Miniconda to C:\Windows\System32\Miniconda.

**Comment by kiwi08522** [comment]
@Totsukawaii I double clicked the install_miniconda.bat file & got the Miniconda folder in my UpdownUnlock directory on my desktop. Then I open up controller & injector as admin, run Lockdown, injector finds Lockdown PID but then Lockdown still closes programs.

**Comment by Totsukawaii** [comment]
> @Totsukawaii I double clicked the install_miniconda.bat file & got the Miniconda folder in my UpdownUnlock directory on my desktop. Then I open up controller & injector as admin, run Lockdown, injector finds Lockdown PID & says it injected successfully but then Lockdown still closes programs. No message box comes up saying Injected.

Wait for miniconda installation to finish before launching the other files. You should get the `press any key to continue` message.

**Comment by skigno** [comment]
I have the same problem, i have done all process correctly, installed miniconda untill "Press any key to continue....." then i run the 2 process as adminstrator controller & injector the controller remanins in Running.... and the injector in waiting Lockdown process.
When i start Lockdown Browser i see the injected window and the Browser told me to kill some opened process, and pressing Ctrl+Up or Down it doesn't work.
There is something i miss? Thanks for help

**Comment by skigno** [comment]
I can't install a previuos version because there is a check to verify the latest version, the actually version installed is the 2.12.009

![Image](https://github.com/user-attachments/assets/3149000c-1d30-46e8-bea0-10e7f59b1826)
